src/data/tags.ts

```diff
@@ -1,3 +1,29 @@
-export const tags = ['JavaScript', 'React', 'Vue', 'Java', 'C#', '.Net'] as const;
+export const tags = [
+  'JavaScript',
+  'TypeScript',
+  'NodeJS',
+  'NestJs',
+  'React',
+  'Vue',
+  'Java',
+  'C#',
+  '.Net',
+  'SRE',
+  'DevOps',
+  'IT Operations',
+  'Scalability',
+  'Software Development',
+  'Software Engineer',
+  'Project Management',
+  'Software Planning',
+  'Solution Architecture',
+  'Software Architecture',
+  'IT Strategy',
+  'Interview',
+  'Team lead',
+  'HR',
+  'Tech recruiter',
+  'educational',
```

Comment:
I don't think this is the right translation. Maybe we could use "Student" for now. What do you think?